spring-vault-core/src/main/java/org/springframework/vault/authentication/AppRoleAuthenticationOptions.java

```diff
@@ -112,6 +143,34 @@ public AppRoleAuthenticationOptionsBuilder path(String path) {
 		}
 
 		/**
+		 * Configure a {@code appRole}.
+		 *
+		 * @param appRole must not be empty or {@literal null}.
+		 * @return {@code this} {@link AppRoleAuthenticationOptionsBuilder}.
+		 */
+		public AppRoleAuthenticationOptionsBuilder appRole(String appRole) {
+
+			Assert.hasText(appRole, "AppRole must not be empty");
+
+			this.appRole = appRole;
+			return this;
+		}
+
+		/**
+		 * Configure a {@code roleToken}.
+		 *
+		 * @param roleToken must not be empty or {@literal null}.
+		 * @return {@code this} {@link AppRoleAuthenticationOptionsBuilder}.
+		 */
+		public AppRoleAuthenticationOptionsBuilder roleToken(String roleToken) {
```

Comment:
Let's rename the token to `initialToken`, like it's named in `CubbyholeAuthentication`.

Reply:
Done